addons/website_slides/models/slide_slide.py

```diff
@@ -396,14 +396,18 @@ def _compute_slides_statistics_category(self, read_group_res):
         """ Compute statistics based on all existing slide categories """
         slide_categories = self.env['slide.slide']._fields['slide_category'].get_values(self.env)
         keys = ['nbr_%s' % slide_category for slide_category in slide_categories]
-        result = dict((cid, dict((key, 0) for key in keys + ['total_slides'])) for cid in self.ids)
+        result = dict((cid, dict((key, 0) for key in keys + ['total_slides', 'completion_time'])) for cid in self.ids)
         for res_group in read_group_res:
             cid = res_group['category_id'][0]
             slide_category = res_group.get('slide_category')
             if slide_category:
                 slide_category_count = res_group.get('__count', 0)
-                result[cid]['nbr_%s' % slide_category] = slide_category_count
+                result[cid]['nbr_%s' % slide_category] += slide_category_count
                 result[cid]['total_slides'] += slide_category_count
+                result[cid]['completion_time'] += res_group['completion_time']
+        for record in self:
+            if not record.is_category:
+                result[record.id]['completion_time'] = record.completion_time
```

Comment:
Do we need this part?

This will just set the completion time on regular slides that already have it, which is strange.

Reply:
Hey :) Because we are computing all of the statistics in the function (both category and not category), we need this part of code so that already computed info about slides isn't lost